load/loader.go

```diff
@@ -4,22 +4,22 @@ import (
 	"bufio"
 	"flag"
 	"fmt"
+	"log"
 	"math"
 	"os"
 	"sync"
 	"sync/atomic"
 	"time"
-	"log"
 )
 
 const (
-	// DefaultBatchSize is the default size of batches to be inserted
+	// defaultBatchSize - default size of batches to be inserted
 	defaultBatchSize = 10000
 	defaultReadSize  = 4 << 20 // 4 MB
 
-	// WorkerPerQueue is the value to have each worker have its own queue of batches
+	// WorkerPerQueue - whether each worker has its own queue of batches
 	WorkerPerQueue = 0
-	// SingleQueue is the value to have only a single shared queue of work for all workers
+	// SingleQueue - whether there is a single shared queue of batches for all workers
```

Comment:
`// SingleQueue is the value for using a single shared queue across all workers`

Reply:
Ok, agreed